website/tools.shtml

```diff
@@ -62,4 +62,11 @@
     can connect to.
 </p>
 
+<h3>Autologin for contestantmachine</h3>
+
+<p> <a href="https://github.com/GEHACK/lightdm-qt5-greeter-ccs-api">LightDM CCS autologin</a> makes it possible
+    to autologin contestantmachines when the contest starts in DOMjudge. This is useful to keep problemdata secret
```

Comment:
isn't to "automatically log into contestant machines" better? Maybe replace contestant with "team". Though I'm fine with this too

Reply:
"contestant machine" should be written separately (in general in English you rarely combine words).